a bunch of revisions and updates for 2.0.0-2.0.2 (#48)
PlayCover/PlayBook: PlayBook/src/getting_started/download_playcover.md

## Prerequisites

> At the moment, PlayCover only works on **Apple Silicon Macs**. The minimum hardware requirement is an Apple M1 chip, PlayCover will also work on all Apple Silicon chips released after the M1.

Reviewer: ```suggestion
PlayCover only works on **Apple Silicon Macs**. The minimum hardware requirement is an Apple M1 chip, but PlayCover will also work on all Apple Silicon chips released after the M1.
```

If you have an Intel Mac, you can explore alternatives like Bootcamp or emulators.

## Download

> You can download the [latest stable release](https://github.com/PlayCover/PlayCover/releases/latest), or build from source by following the instructions below. 

Reviewer: This ‘below’ seems to be unintended

Author: it's way below the page i think lol

Reviewer: They aren’t since it seems to refer to an old version of the document. Building instructions have since been moved to another article.

Author: Should i just make it link to this page then? https://docs.playcover.io/building_from_source/install_prerequisites

Author: i thought the homebrew cask thing was building from source

> Additionally, you can download the [latest nightly version](https://nightly.link/PlayCover/PlayCover/workflows/2.nightly_release/develop/PlayCover_nightly_210.dmg) to test out experimental features not yet added to the stable release.

Reviewer: https://nightly.link/playcover/playcover/workflows/2.nightly_release/develop is better instead.
